src/site/stages/build/options.js

```diff
@@ -147,26 +150,69 @@ function deriveHostUrl(options) {
   ];
 }
 
-function setUpFeatureFlags(options) {
+// Sets up the CMS feature flags by either querying the CMS for them
+// or using ../../utilities/featureFlags
+async function setUpFeatureFlags(options) {
   global.buildtype = options.buildtype;
-  const {
-    enabledFeatureFlags,
-    featureFlags,
-  } = require('../../utilities/featureFlags');
+  let enabled;
+
+  if (shouldPullDrupal(options)) {
+    const apiClient = getDrupalClient(options);
+    const result = await apiClient.proxyFetch(
+      `${apiClient.getSiteUri()}/flags_list`,
+    );
+
+    const flags = (await result.json()).data;
+    // eslint-disable-next-line no-console
+    console.log('Drupal feature flags:\n', flags);
+
+    // Using a Proxy to throw an error during the build if the feature
+    // flag referenced isn't returned from Drupal.
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy
```

Comment:
This is where things get a little weird. If the comments don't explain what's happening sufficiently, please call it out; our future selves (and any new engineers) need to know why this is here six months from now.